portfolio/src/main/webapp/index.html

```diff
@@ -74,6 +74,28 @@ <h2>More about me</h2>
             <p>I am not just my heritage, I am a young man with a goal to have others understand who I am and surround myself with people who will support me and challenge me. Education is what is going to help me achieve understanding, but I believe it is through the process that I will gain the relationships and knowledge that will lead to creating the network of people I strive to be a part of. That’s why I was so excited to go to college: because one of the fundamental functions of college is to bring a many of these kinds of people together. It gives me the ability to actualize my aspirations.
               </p>
             </div>
+          <div class = "topic">
+              <h2>Comments</h2>
+              <div id = "comments-container"></div>
+              <form method="GET">
+                <p>How many comments would you like to view?(1-10)</p>
+                <input type="number" name="user-comment-num" min="1" max="10" value="5">
+                <input type="submit" onclick="getComments()" />
+                </form>
+              <form mehtod="POST">
+                  <p>Would you like to burn the comment section to the ground?<p>
+                  <button type="submit" onclick="deleteAllComments()">Delete All Comments</button>
+                  </form>
+              </div>
+          <div class="topic">
+            <h2>Add a Comment</h2>
+            <form action="/data" method="POST">
+              <input name="name-input">Name</input>
```

Comment:
To use a default value for `<input>`, you may want to use:
`<input name="name-input" value="Name">`

see: https://www.w3schools.com/tags/att_input_value.asp